src/main/java/com/rultor/agents/github/CommentsTag.java

```diff
@@ -43,10 +43,16 @@
 import com.rultor.agents.daemons.Home;
 import java.io.IOException;
 import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
 import java.util.Date;
+import java.util.List;
 import java.util.ResourceBundle;
+import java.util.regex.Pattern;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
 import org.xembly.Directive;
```

Comment:
@gumbelmj I know this class suppresses `ClassDataAbstractionCouplingCheck` as well as uses `@SuppressWarnings("PMD.ExcessiveImports")`, **but** this should be seen as a pointer towards improving both issues, not further exacerbate them!